docs/api.rst

```diff
@@ -286,7 +286,8 @@ at
 
 .. code-block:: haskell
 
-  at :: Time -> a -> Stream a
+  at :: (Time, a) -> Stream a
+  at :: Time -> Stream void
```

Comment:
Unfortunately, the second signature is only valid for JavaScript.  The TS types currently prevent calling `at` with just 1 argument.

Maybe we can keep only the first signature, and add a note for JavaScript users?

```suggestion
```